bin/sync.sh

```
@@ -0,0 +1,47 @@
+#!/bin/sh
+set -x
```

I would use -e here, because the final rm hides the possible errors in the first steps.